bindings/mnt/pyfiction/include/pyfiction/algorithms/simulation/sidb/physically_valid_parameters.hpp

```diff
@@ -37,21 +37,20 @@ inline void physically_valid_parameters(pybind11::module& m)
 {
     namespace py = pybind11;
 
-    py::class_<fiction::operational_domain<fiction::parameter_point, uint64_t>>(m, "physically_valid_parameters_domain")
+    py::class_<fiction::sidb_simulation_domain<fiction::parameter_point, uint64_t>>(
```

Comment:
Missing docstrings for the class and member functions

Reply:
The problem is that the docstring is not generated. Do you have an idea why?

Reply:
Unfortunately not. I have observed this behavior a few times myself as well. It's really mysterious to me :thinking:

Reply:
okay, can we merge it anyway?